geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,131 @@
+"""LogNormal Sampler"""
+
+import geomstats.backend as gs
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.spd_matrices import SPDMatrices
+
+
+class LogNormal:
+    """LogNormal Sampler on manifold of SPD Matrices and Euclidean Spaces.
+
+    (1)For Euclidean Spaces, if X is distributed as Normal(mean, cov), then
+    exp(X) is distributed as LogNormal(mean, cov).
+
+    (2)For SPDMatrices, LogNormal distribution is defined by embedding mean
```

Comment:
For SPDMatrices, the LogNormal distribution is defined by transforming the mean -- an SPD matrix -- into a symmetric matrix through the Matrix logarithm, which gives the element "log-mean" that now belongs to a vector space. This log-mean and given cov are used to parametrize a Normal Distribution.

Reply:
> For SPDMatrices, the LogNormal distribution is defined by transforming the mean -- an SPD matrix -- into a symmetric matrix through the Matrix logarithm, which gives the element "log-mean" that now belongs to a vector space. This log-mean and given cov are used to parametrize a Normal Distribution.

Done.